react-accordion-component/src/index.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Accordion from './accordion';
+ReactDOM.render(
```

Comment:
Here you should have an array of topics defined. You then pass the array of topics into the `Accordion` component:

```JSX
<Accordion topics={topics} />
```